staging/src/k8s.io/kube-controller-manager/config/v1alpha1/types.go

```diff
@@ -179,6 +179,9 @@ type AttachDetachControllerConfiguration struct {
 	// ReconcilerSyncLoopPeriod is the amount of time the reconciler sync states loop
 	// wait between successive executions. Is set to 60 sec by default.
 	ReconcilerSyncLoopPeriod metav1.Duration
+	// Flag to disable force detach if the maximum unmount time is exceeded.
+	// Is false by default, and thus force detach on unmount is enabled.
```

Comment:
```suggestion
	// disableForceDetachOnTimeout disables force detach when the maximum unmount time is exceeded.
	// Is false by default, and thus force detach on unmount is enabled.
```

Since, here, this is a component config field, not a flag.

@msau42 Does this follow API review guidelines?  I can't remember if want "disable" fields or if we prefer "enable" fields that's can be used to disable functionality.  Note that we already have `DisableAttachDetachReconcilerSync` as a sibling field.